commonknowledge/meep-intelligence-hub
# Set env variables for sentry and posthog

## Description
This PR 
- adds environments to Sentry and Posthog using the NODE_ENV variable that Next.js sets
- stops sending posthog events and sentry errors to dashboards in development
- Makes NEXT_PUBLIC_SENTRY_DSN more secure by changing to SENTRY_DSN 

Question for @janbaykara - should POSTHOG_HOST and POSTHOG_KEY variables be NEXT_PUBLIC? Is this secure?

## Motivation and Context
Addresses issue [MEEP-300](https://linear.app/commonknowledge/issue/MEEP-300/distinguish-production-from-staging-in-sentry)

## How Can It Be Tested?
Download the branch and run locally
Make sure environment variables are set as per the .env.example files. 
Generate some errors
Observe that they are logged to the console and not sent to Sentry dashboard
Generate some page view events (by clicking around on pages)
Observe that they are not sent to Posthog dashboard

## How Will This Be Deployed?
Need to change NEXT_PUBLIC_SENTRY_DSN to SENTRY_DSN on Render
I've regnerated the SENTY_DSN and updated it in Render

## Types of changes
